DESCRIPTION

```diff
@@ -87,7 +87,7 @@ Suggests:
     htmltools,
     rmarkdown,
     testthat,
-    RSelenium, XML,
```

Comment:
We also do not need to start selenium docker in tests.yaml. I think the build is currently failing on Github because of that

Reply:
ok fixed